src/main/scala/tech/cryptonomic/conseil/util/JsonUtil.scala

```diff
@@ -44,4 +46,20 @@ object JsonUtil {
   def fromJson[T](json: String)(implicit m : Manifest[T]): T = {
     mapper.readValue[T](json.filterNot(Character.isISOControl))
   }
+
+  /** extractor object to read accountIds from a json string, based on the hash format*/
+  object AccountIds {
+    /** regular expression matching a valid account hash as a json string */
+    val AccountHashExpression: Regex = """"(tz[1-3]|KT1)[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{33}"""".r
```

Comment:
is it necessary to write all of the digits/letters? `[a-zA-Z0-9]` won't work?

Reply:
I thought the same at first, but not all are included
Look at this crazy thing here: https://en.bitcoin.it/wiki/Base58Check_encoding